SpatialGDK/Source/SpatialGDK/Private/Schema/UnrealObjectRef.cpp

```diff
@@ -133,6 +133,61 @@ FUnrealObjectRef FUnrealObjectRef::FromObjectPtr(UObject* ObjectValue, USpatialP
 	return ObjectRef;
 }
 
+FUnrealObjectRef FUnrealObjectRef::FromSoftObjectPtr(const FSoftObjectPtr& ObjectPtr)
```

Comment:
Do we prefer this or constructor overrides?

Reply:
Following the trend from FromObjectPtr/ToObjectPtr. I'm fine with that version since it separates storage from initialization.